packages/sites/src/index.ts

```diff
@@ -1,7 +1,7 @@
 /* Copyright (c) 2018 Environmental Systems Research Institute, Inc.
  * Apache-2.0 */
 
-export * from "./domains";
+export * from "@esri/hub-common";
```

Comment:
Something about this... seems problematic... this will export all of hub common in hub-sites... is there some way to scope this?

Reply:
oh yeah, this is bad - I suspect @andrewctate was just doing a global find and replace on `from "./domains"`. Just need to undo that in this instance

Reply:
Thanks for noting this. Definitely unintentional.